app/views/container_group/_main.html.haml

```diff
@@ -1,7 +1,10 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
+    :items => textual_group_properties}
```

Comment:
I think that is much clearer.  I would vertically align the `=>` in `:partial =>` and `:locals =>`.